### PR TITLE
Update contribute.md

### PR DESCRIPTION
**Pull request by ebolch**
Fixed typos and clarified some instructions

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -1,84 +1,93 @@
 # Contributing to this Repository
 
+> Please submit a pull request early in the development phase, outlining the changes you intend to make or features you intend to add. This allows us to offer feedback early on, ensuring your contribution can be added to the repository before you invest a significant amount of time.
```
Comment (amfriesz) [inline CONTRIBUTE.md:3]: @ebolch, do we want them to submit a PR or should that information be added to an Issue?

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -1,84 +1,93 @@
...
 
     ```
     git remote add upstream original-repository-url
     ```
 
-    - Using `git remote -v` will show two remote repositories named:  
-        - `upstream`, which refers to the repository  
-        - `origin`, which refers to your personal fork  
-    - Often, updates to an `upstream` repository will occur while you are developing changes on your personal fork. You can pull the latest changes from `upstream`, including tags:
+    - You can use to following to view the remote repositories:
```
Comment (amfriesz) [inline CONTRIBUTE.md:41]:
```suggestion
    - You can use the following to view the remote repositories:
```

File: CONTRIBUTE.md [hunk 3]
```diff
@@ -1,84 +1,93 @@
...
 
     ```git
     git push origin feature-name
     ```
 
     - Enter username and password, depending on your settings, you may need to use a [Personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
-    - Make a Pull Request. On the GitHub page for your personal fork, there will be a green Pull Request button.    -
 
-## Adding New Notebooks or Example Workflows
+5. To submit your contribution, make a pull request to the `dev` branch our repository on the GitHub page. There will be a green Pull Request button.
```
Comment (amfriesz) [inline CONTRIBUTE.md:84]: @ebolch, Submit to the 'dev' branch? Are we planning to maintain two identical branches?

File: user_contributed/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# User Contributed Materials
+
+This directory is a location that users can share new code, notebooks, or examples from their research. To contribute please follow the instructions outlined in the the [Adding New Notebooks or Example Workflows Section](https://github.com/nasa/VITALS/blob/main/CONTRIBUTE.md#adding-new-notebooks-or-example-workflows) of our [contributing guide](https://github.com/nasa/VITALS/blob/main/CONTRIBUTE.md).
```
Comment (amfriesz) [inline user_contributed/README.md:3]:
```suggestion
This directory is a location that users can share new code, notebooks, or examples from their research. To contribute please follow the instructions outlined in the [Adding New Notebooks or Example Workflows Section](https://github.com/nasa/VITALS/blob/main/CONTRIBUTE.md#adding-new-notebooks-or-example-workflows) of our [contributing guide](https://github.com/nasa/VITALS/blob/main/CONTRIBUTE.md).
```

File: CONTRIBUTE.md (hunk 1 above)
Comment (ebolch) [reply CONTRIBUTE.md:3]: I think having a contributor submit a PR will make it easier to review

**Pull request merged**